html-lists/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ordered and unordered lists</title>
+  </head>
+  <body>
+    <h1 style="text-transform: uppercase">
+      html <span style="text-transform: capitalize">lists</span>
+    </h1>
+
+    <h2><strong>Ordered list example</strong></h2>
```

Comment:
`h2` will be bold by default, and is already semantic, so no need to include the `strong` element